### PR TITLE
Changed errpacture and logcapture method names to be cleaner.

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: Bartlomiej Plotka <bwplotka@gmail.com>

cc @kakkoyun @GiedriusS does it make sense?

### CODE REVIEW

**Review by kakkoyun — APPROVED** [review]
💯

File: core/pkg/errcapture/do.go [hunk 1]
```diff
@@ -11,24 +11,36 @@ package errcapture
...
-	*err = merrors.New(*err, errors.Wrapf(closer(), format, a...)).Err()
+func Do(err *error, doer doFunc, format string, a ...interface{}) {
+	derr := doer()
+	if err == nil {
+		return
+	}
+
+	// For os closers, it's a common case to double close. From reliability purpose this is not a problem it may only indicate
+	// surprising execution path.
+	if errors.Is(derr, os.ErrClosed) {
```
Comment (GiedriusS) [inline core/pkg/errcapture/do.go:32]: It's weird that now these functions are kind of "generic" but they have these specific bits in them. Perhaps it should accept a slice of errors that would be ignored?

**Review by GiedriusS — COMMENTED** [review]
I guess it makes sense, just one minor comment.

File: core/pkg/errcapture/do.go (hunk 1 above)
Comment (bwplotka) [reply core/pkg/errcapture/do.go:32]: Agree, is just a very common error that is breaking idempotency, so not sure 🤔

**Pull request merged**